app/models/record_mailer.rb

```diff
@@ -17,6 +17,16 @@ def email_record(documents, details, url_gen_params)
     mail(to: details[:to], subject: subject)
   end
 
+  def email_report_issue(_documents, details, url_gen_params)
```

Comment:
`_documents` isn't being used in this method. Rubocop, I bet, put that underscore in there automatically. That's a Ruby-ism for indicating that a parameter is being passed in but not used. So you should be able to delete it and just have:

``` ruby
def email_report_issue(details, url_gen_params)
...
```